recipe/meta.yaml

```diff
@@ -1,6 +1,6 @@
 package:
   name: conda-forge-build-setup
-  version: 4.4.17
+  version: 4.4.18
```

Comment:
In retrospect, it might have been better to do `5.0.0` given the bootstrapping issue. Though it's not that big of a deal. We will likely go to `5.0.0` soon enough with `conda-build`.